x-pack/plugins/observability/server/services/slo/sli_client.ts

```diff
@@ -56,9 +54,7 @@ export class DefaultSLIClient implements SLIClient {
       return indicatorDataBySlo;
     }
 
-    const result = await this.esClient.msearch<unknown, Record<AggKey, AggregationsSumAggregate>>({
-      searches,
-    });
+    const result = await this.esClient.msearch({ searches });
```

Comment:
💬 I'm going to refactor this client in a later PR. The initial design of fetching the IndicatorData and then computing things (sli value, error budget, etc..) worked well until now. 
I think this client should compute the summary directly, like we do in the historical summary client.